bench/db_bench.ml

```diff
@@ -116,8 +95,8 @@ module Index = struct
     in
     let ratio_reads = float_of_int stats.nb_reads /. float_of_int nb_entries in
     Log.app (fun l ->
-        l "\tread amplification in bytes = %f; in nb of reads = %f "
-          ratio_bytes ratio_reads)
+        l "\tread amplification in bytes = %f; in nb of reads = %f " ratio_bytes
+          ratio_reads)
```

Comment:
Are you using the right version of OCamlformat?

Reply:
Yep! 0.12